src/ddmd/mtype.d

```diff
@@ -3091,11 +3098,13 @@ extern (C++) abstract class Type : RootObject
             }
             if (t.iscomplex())
             {
-                fprintf(global.stdmsg, "%s: use of complex type '%s' is scheduled for deprecation, use 'std.complex.Complex!(%s)' instead\n", p ? p : "", toChars(), rt.toChars());
+                deprecation(loc, "use of complex type `%s` is scheduled for deprecation, use `std.complex.Complex!(%s)` instead\n",
```

Comment:
It's clearly not `scheduled` for deprecated, it is deprecated, as it calls `deprecation()`